File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let yArr = []
    let xArr = []
    for (let y = 0; y < wallpaper.length; y++){
        for (let x = 0; x < wallpaper[0].length; x++){
            if (wallpaper[y][x] === "#"){
                yArr.push(y)
                xArr.push(x)
            }
        }
    }
    return [Math.min(...yArr), Math.min(...xArr), Math.max(...yArr)+1, Math.max(...xArr)+1];
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    yArr = []
    xArr = []
    
    for y in range(len(wallpaper)):
        for x in range(len(wallpaper[0])):
            if wallpaper[y][x] == "#":
                yArr.append(y)
                xArr.append(x)
                
    return [min(yArr),min(xArr),max(yArr)+1,max(xArr)+1]